webui/tests/test-cluster/master.yaml

```diff
@@ -11,7 +11,7 @@ checkpoint_storage:
 
 db:
   user: postgres
-  host: localhost
```

Comment:
This is more curiosity than anything, but why does this have to change?

Reply:
I think a short version would be less chance for DNS (or hosts entry?) to mess up and delay responding with the address.

https://determined-ai.slack.com/archives/CSKCTUKEE/p1611877804019800?thread_ts=1611864602.016200&cid=CSKCTUKEE